recipes/pyramid/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "pyramid" %}
+{% set version = "1.10.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 37c3e1c9eae72817e0365e2a38143543aee8b75240701fa5cb3a1be86c01a1c0
+
+build:
+  noarch: python
```

Comment:
Pyramid is [>py34 only](https://github.com/Pylons/pyramid/blob/master/setup.py#L98). To avoid Python 2.7 users to install this package, use `python >=3.4` in `host` and `run`.

Reply:
Actually you are looking on master which is a dev version. The current 1.10.1 has support.

Reply:
Ahh indeed. 👍 

I suggest to add a comment next to the version:

```
  version: "{{ version }}"
  # when updating to 2.0, make sure to add "python >=3.4" to "host" and "run" sections
  # because 2.0 no longer supports Python 2
```

To make sure we do the proper requirement change when the time comes. 😁 

(I don't know if they plan to release `master` as 2.0 for a fact, just assuming)